### PR TITLE
[CI] Add Pin to Crust workflow

### LINKED ISSUE #1339: [CI] Uniswap Interface decentralized IPFS pin
## Is your feature request related to a problem? Please describe.

Encouraged by https://uniswap.org/blog/ipfs-uniswap-interface/, we would like to contribute and provide a decentralized IPFS pin solution for Uniswap Interface, to further increase the availability and decentralization.

## Describe the solution you'd like

We have built a standard Github Workflow to pin a website to Crust Network in a decentralized way: https://github.com/crustio/ipfs-crust-pinner. And specifically for Uniswap Interface, we have created a workflow to pin it: https://github.com/crustio/ipfs-crust-pinner/actions/workflows/uniswap-interface.yml.

We suggest a code change after https://github.com/Uniswap/uniswap-interface/blob/main/.github/workflows/release.yaml#L48-L55, to include [ipfs-crust-pinner](https://github.com/crustio/ipfs-crust-pinner) action. It will distribute Uniswap Interface over Crust Network and get 50+ IPFS replicas. We can help prepare a code PR to do it.

Crust Network (https://crust.network/) is a decentralized storage network based on IPFS and Substrate, and it basically provides decentralized IPFS Pin service. Right now Crust is running preview network `Maxwell` and got 1000+ nodes joined in to provide IPFS storage (https://telemetry.polkadot.io/#list/Crust%20Maxwell). Crust is open source(https://github.com/crustio/crust).

## Describe alternatives you've considered

N/A

## Additional context

You can contact us at hi@crust.network. We can help prepare a code PR. The pin service will be free for Uniswap, and you can mail us for necessary seeds to call Crust IPFS Pin service.

### PR DESCRIPTION
**Pull request by zikunfan**
Resolve #1339

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -54,6 +54,14 @@ jobs:
           pinata-api-key: ${{ secrets.PINATA_API_KEY }}
           pinata-secret-api-key: ${{ secrets.PINATA_API_SECRET_KEY }}
 
+      - name: Pin to Crust
+        uses: crustio/ipfs-crust-action@v1.0.8
+        continue-on-error: true
+        timeout-minutes: 2
+        with:
+          cid: ${{ steps.upload.outputs.hash }}
```
Comment (moodysalem) [inline .github/workflows/release.yaml:62]: Does this mean the pin is fetched from other ipfs nodes?

Comment (zikunfan) [reply .github/workflows/release.yaml:62]: Yes. The pin will be fetched from other (50+) IPFS nodes. The whole process after this PR will shows like below:
image

PS: the IPFS nodes status of Crust Network can be checked [here](https://telemetry.polkadot.io/#map/Crust%20Maxwell). **You may need Crust seeds to call the action. You can send a mail to hi@crust.network and I will send it over.**

Comment (zikunfan) [reply .github/workflows/release.yaml:62]: Hey @moodysalem , any questions, please feel free to ask me

**Comment by zikunfan** [comment]
`secrets.CRUST_SEEDS` already send by email.

**Pull request merged**